docs/development/workflow/get_devel_version.rst

```diff
@@ -222,30 +222,43 @@ of `Astropy`_, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
+    There are some circumstances in which this quick method of
     activating your copy of `Astropy`_ will NOT work and you need to go
-    straight to using a virtual python environment:
+    straight to using a virtual python environment, for example:
 
-    + You use Python 3.
     + You want to work on C or Cython code in `Astropy`_.
 
+If you have decided to use the recommended "activation" method with pip, 
+please note the following: Now is the time to check that you have all 
+the required dependencies installed on your machine to make sure 
+the proposed pip install would be a successful one. Namely, as 
+you are effectively building from source, please check that you have both "cython"
+as well as "jinja2" installed using pip. (Note: Using conda to install these two 
+dependencies would normally not work and would result in error(s) thrown.) 
+Please note that by pip we really mean the python 3 version of pip. 
+If you have any problem with different versions of pip installed,
+try aliasing to resolve the issue. If you are unsure about which pip version you 
+are using, try the command ``which python`` on the terminal. 
 In the directory where your copy of `Astropy`_ is type::
 
     pip install -e .[all,test]
```

Comment:
I think this is actually unclear; should the user copy/paste `[all, test]` - I would just remove that, and show it later.

Reply:
Yup, removed... I think I can leave it out since testing is discussed in a section that follows this.